app/src/main/res/values/colors.xml

```diff
@@ -244,12 +246,15 @@
 
     <!-- Bookmark buttons -->
     <color name="bookmark_favicon_background">#DFDFE3</color>
-
+  
```

Comment:
nit - we can remove these extraneous changes

Reply:
I actually don't know how that happened. There was one line between them and currently it is also.
I don't know how to fix it.
![resim](https://user-images.githubusercontent.com/17825767/85685914-2e0cbf80-b6d8-11ea-988f-302332785464.png)

Reply:
Fixed this but now with styles line 78